Library/Formula/robot-framework.rb

```diff
@@ -0,0 +1,62 @@
+class RobotFramework < Formula
+  desc "Open source test automation framework for acceptance testing"
+  homepage "http://robotframework.org/"
+  url "https://github.com/robotframework/robotframework/archive/3.0.tar.gz"
+  sha256 "1b830fa5e4470ff6b2d404bd99b357cfce4a2abd15c255373045a77d44e517a5"
+
+  head "https://github.com/robotframework/robotframework.git", :branch => "master"
+  depends_on :x11
```

Comment:
If this is a hard dependency this will need to be submitted to homebrew/homebrew-x11 instead, sorry!